src/TRoom.cpp

```diff
@@ -1632,23 +1632,25 @@ void TRoom::auditExit(int& exitRoomId,                     // Reference to where
             QString auditKeyLocked = qsl("audit.invalid_exit.%1.isLocked").arg(dirCode);
             userData.insert(auditKeyLocked, qsl("true"));
             if (mudlet::self()->showMapAuditErrors()) {
-                infoMsg.append(tr("\nIt was locked, this is recorded as user data with key:\n"
-                                  "\"%1\".")
-                                       .arg(auditKeyLocked));
+                infoMsg.append(
+                    qsl("\n%1").arg(
+                        tr("It was locked, this is recorded as user data with key:\n\"%1\".")
+                        .arg(auditKeyLocked)));
             }
-            logMsg.append(tr(R"(  It was locked, this is recorded as user data with key: "%1".)").arg(auditKeyLocked));
+            logMsg.append(qsl("  %1").arg(tr(R"(It was locked, this is recorded as user data with key: "%1".)").arg(auditKeyLocked)));
```

Comment:
:-1: This is an additional text sentence that may (or may not) be appended to others to form a paragraph. However not every locale has a space (or two) between sentences (e.g. Chinese, Japanese) but this construct is now forced upon them.

Reply:
This whole section is ripe for a rewrite (but my fu isn't strong enough, yet) as it duplicates many texts almost (except for formatting) 1:1 for inclusion in two different bodies, but these texts should come from a singular source that can be translated once, and formatted automatically as each locale sees fit. Now let's see if Chinese or somebody finds trouble with this. We don't even translate to Japanese today, so let's cross that bridge when we get there.

Reply:
> This whole section is ripe for a rewrite (but my fu isn't strong enough, yet) as it duplicates many texts almost (except for formatting) 1:1 for inclusion in two different bodies ...

Yes, the duplication and the formatting differences are precisely because of the differences in those two bodies. One  is organised to collate all the different pieces information for a particular room or area in one place (in the `errors.txt`) file - and **will not** contain the ID or name because that will be the separator/header whereas the other may be displayed on the main console as it is discovered and ***does*** have to identify which room or area it effects. As such it is hard for me to see how they could be derived from a common text AND still read clearly.

According to https://en.wikipedia.org/wiki/Chinese_punctuation#Spacing spaces are not used between words in (Simplified, i.e. China) Chinese.